client/FxAccountClient.js

```diff
@@ -1031,4 +1099,30 @@ define([
   return FxAccountClient;
 });
 
+var METRICS_CONTEXT_OPTIONS = {
```

Comment:
nit: all the new stuff here, could it be in its own file `metrics.js` or `metrics-contexts.js` or something? 
With the goal of keeping this file cleaner.  Up to you @philbooth @shane-tomlinson